nemo-physical/src/function/definitions/numeric.rs

```diff
@@ -242,6 +296,83 @@ impl BinaryFunction for NumericDivision {
     }
 }
 
+/// Bitwise and
+///
+/// For a list of integers,
+/// returns the integer resulting from perfoming an "and" on their bit representation.
+///
+/// Returns `None` if the input parameters are not integers or no input parameters are given.
+#[derive(Debug, Copy, Clone)]
+pub struct BitAnd;
+impl NaryFunction for BitAnd {
+    fn evaluate(&self, parameters: &[AnyDataValue]) -> Option<AnyDataValue> {
+        if let Some(list) = NumericList::from_any_list(parameters) {
+            match list {
+                NumericList::Integer(integers) => numeric_bitwise_and(&integers),
+                _ => None,
+            }
+        } else {
+            None
+        }
```

Comment:
Similar to above, this could use `Option::and_then`.
This actually applies to most (or even all) of the added/changed functions below.